package.json

```diff
@@ -2,21 +2,44 @@
   "name": "mindlogger-rebuild",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
+  "type": "commonjs",
   "scripts": {
     "dev": "vite",
```

Comment:
Please add "npx" for all commands (example "npx vite") that refer to an npm package that is located in devDependencies to ensure the build will not fail during CI/CD processes.

Reply:
Done